### PR TITLE
Added license header

### PR DESCRIPTION
**Pull request by lslezak**
Needed to pass the `rake check:license` check.

### CODE REVIEW

**Comment by imobach** [comment]
LGTM

File: test/migration_proposal_client_test.rb [hunk 1]
```diff
@@ -1,3 +1,22 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
```
Comment (kobliha) [inline test/migration_proposal_client_test.rb]: bad bad :)

Comment (kobliha) [inline test/migration_proposal_client_test.rb]: Copyright (c) 2012-2014 Novell, Inc.
Copyright (c) 2014-2015 SUSE LLC

**Comment by kobliha** [comment]
Nice, let's reuse @imobach 's LGTM :)

**Pull request merged**